test/integration/server.h

```diff
@@ -373,7 +373,9 @@ class TestIsolatedStoreImpl : public StoreRoot {
   void setHistogramSettings(HistogramSettingsConstPtr&&) override {}
   void initializeThreading(Event::Dispatcher&, ThreadLocal::Instance&) override {}
   void shutdownThreading() override {}
-  void mergeHistograms(PostMergeCb) override {}
+  void mergeHistograms(PostMergeCb cb) override { merge_cb_ = cb; }
+
+  PostMergeCb merge_cb_;
```

Comment:
nit: expose runMergeCallback() as a public function and leave `merge_cb_` private.